### PR TITLE
Catch exceptions when no VMs running in GCE

### PR DESCRIPTION
**Pull request by hyunwoo18**

### CODE REVIEW

File: src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py [hunk 1]
```diff
@@ -41,8 +41,15 @@ def transform(self, data_block):
             it = row["InstanceType"]
             entry_name = row["EntryName"]
 
-            occupancy_df = gce_occupancy[((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))]
-            occupancy = float(occupancy_df["Occupancy"].values[0]) if not occupancy_df.empty else 0
+            try:
+                occupancy_df = gce_occupancy[
+                    ((gce_occupancy.AvailabilityZone == az) & (gce_occupancy.InstanceType == it))
+                ]
+            except:
```
Comment (mambelli) [inline src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py:48]: Do you know which exceptions can be risen? Is it possible to catch specific exceptions?

File: src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py [hunk 2]
```diff
@@ -41,8 +41,15 @@ def transform(self, data_block):
...
+                occupancy = 0
+                self.logger.debug(f"GceFigureOfMerit transform: Looks like no VMs runnig in GCE.")
```
Comment (mambelli) [inline src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py:50]: If you don't include any variable there is no need for `f""`, just the string is OK `""`.

**Review by mambelli — CHANGES_REQUESTED** [review]

**Comment by hyunwoo18** [comment]
Marco, I updated the branch to implement your comments.
I am not sure if you received notification from GH about this.

File: src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py (hunk 1 above)
Comment (hyunwoo18) [reply src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py]: This is taken care of.

File: src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py (hunk 2 above)
Comment (hyunwoo18) [reply src/decisionengine_modules/GCE/transforms/GceFigureOfMerit.py]: This is also taken care of. 
(I just forgot to erase f earlier)

**Review by mambelli — APPROVED** [review]

**Comment by mambelli** [comment]
There is still some problem here, GH says "This branch cannot be rebased due to conflicts". Will look into it later. I may have to cherry-pick the changes

**Comment by hyunwoo18** [comment]
Yes, when I was trying a rebase, there was a message that said to resolve a conflict in 
GCE/transforms/GceFigureOfMerit.py.
I resolved the conflict and proceeded with 
git rebase --continue
git add GCE/transforms/GceFigureOfMerit.py
and pushed to origin/t381.

But I don't understand what more GH system is complaining about.
I just followed what GH recommended.

Thinking again, maybe I should have just updated my code to reflect your change 
in the same file, GCE/transforms/GceFigureOfMerit.py
instead of rebasing..

**Comment by mambelli** [comment]
Replaced by PR #413

**Pull request closed without merging**